discovery/consul/consul.go

```diff
@@ -100,6 +102,10 @@ type SDConfig struct {
 	Username     string             `yaml:"username,omitempty"`
 	Password     config_util.Secret `yaml:"password,omitempty"`
 
+	// Only scrap services nodes this healthy, inclusive. Possible values are "critical", "warning",
+	// "passning".
+	// Defaults to "citical" (all services nodes) if empty.
+	MinHealth string `yaml:"min_health,omitempty"`
```

Comment:
This can be done with relabelling, and it is recommended to scrape nodes that are apparently unhealthy.